client/my-sites/themes/themes-toolbar-group/index.tsx

```diff
@@ -29,6 +29,7 @@ const ThemesToolbarGroup: React.FC< ThemesToolbarGroupProps > = ( {
 			initialActiveIndex={ activeIndex }
 			forceSwipe={ 'undefined' === typeof window }
 			onClick={ ( index: number ) => onSelect( items[ index ]?.key ) }
+			swipeBreakpoint="<60px"
```

Comment:
This is the only way I found to avoid the component from becoming a horizontal scroll. If you know a different way I'm open to changing this.

Reply:
Maybe we could add a new optional prop to the `ResponsiveToolbarGroup` component to disable the swipe entirely?

Reply:
I was trying to avoid changing the component, but it seems there isn't any way around it...

Changed the component.